src/main/webapp/homepage_script.js

```diff
@@ -45,9 +45,9 @@ if (document.readyState === 'loading') {
 
 /* Function adds all the necessary UI 'click' event listeners*/
 function addUIClickHandlers() {
-    // adds showCreateTaskModal and closeCreateTaskModal click events for the add task button
-    if (document.body.contains(document.getElementById("addtaskbutton"))) {
-        document.getElementById("addtaskbutton").addEventListener("click", showCreateTaskModal);
+    // adds showModal and closeModal click events for the add task button
```

Comment:
Since the event listeners are `showCreateTaskModal` and `closeCreateTaskModal`, should the comments be "adds 'showCreateTaskModal' and 'closeCreateTaskModal' click events for the add task button" instead?

Reply:
Ah yes, I forgot to edit the comment, I'll change this now, thanks!